essentials/Albert/install.sh

```diff
@@ -0,0 +1 @@
+curl "https://build.opensuse.org/projects/home:manuelschneid3r/public_key" | sudo apt-key add -
```

Comment:
Shouldn't `sudo apt update; sudo apt install albert` be here?

Reply:
Oh thank you. Have tested on 3 different Pi's. Something must have gone wrong while copying back and forth.... i'll fix it.